clap-utils/src/input_validators.rs

```diff
@@ -6,50 +6,86 @@ use solana_sdk::{
     pubkey::Pubkey,
     signature::{read_keypair_file, Signature},
 };
+use std::fmt::Display;
 use std::str::FromStr;
 
+fn is_parsable_generic<U, T>(string: T) -> Result<(), String>
```

Comment:
@svenski123 Nice! Could you split off this generic `clap-utils` parsing message improvement as a separate pr? I think this is useful for v1.1 as well and we want to manage logically-separatable changes as separate PRs. Like #10096, I'll land that pr first.

Reply:
Done in #10599, will rebase this once that's been merged.